redash/serializers/__init__.py

```diff
@@ -246,6 +246,7 @@ def serialize_dashboard(obj, with_widgets=False, user=None, with_favorite_state=
     d = {
         "id": obj.id,
         "slug": obj.slug,
+        "name_as_slug": obj.name_as_slug,
```

Comment:
Why not return it as the slug value?

Reply:
Now that we migrated all API routes to use an id, we can drop this?

Reply:
Done 👍